### ISSUE TITLE
Uninstalling a LV2 plugin causes system to hand loading snapshot that uses it.

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#1110]
# Describe the issue
If a LV2 plugin is used in a snapshot then the LV2 plugin is uninstalled, the snapshot will fail to load with errors that cause Zynthian to lockup.
# To Reproduce
## Steps to reproduce the behavior:
1. Create a snapshot with LV2 plugin
2. Unistall the LV2 plugin
3. Load snapshot
 ## Expected behaviour
Snapshot loads without the plugin. Ideally it shows a warning about the missing plugin(s).
## Actual behaviour
Permamanet loading screen with message, "ERROR: Invalid snapshot".
# Additional context
Log:
```
Jul 04 11:23:32 zynthian-rpi5 startx[1585]: ERROR:zynthian_chain_manager.add_processor: Engine 'JV/Outotune' not found!
Jul 04 11:23:32 zynthian-rpi5 startx[1585]: ERROR:zynthian_state_manager.load_snapshot: Invalid snapshot: list index out of range
Jul 04 11:23:32 zynthian-rpi5 startx[1585]: Traceback (most recent call last):
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:   File "/zynthian/zynthian-ui/zyngine/zynthian_state_manager.py", line 1014, in load_snapshot
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:     self.load_zs3("zs3-0")
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:   File "/zynthian/zynthian-ui/zyngine/zynthian_state_manager.py", line 1247, in load_zs3
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:     chain.rebuild_graph()
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:   File "/zynthian/zynthian-ui/zyngine/zynthian_chain.py", line 412, in rebuild_graph
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:     self.rebuild_audio_graph()
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:   File "/zynthian/zynthian-ui/zyngine/zynthian_chain.py", line 325, in rebuild_audio_graph
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:     for source in self.audio_slots[self.fader_pos - 1]:
Jul 04 11:23:32 zynthian-rpi5 startx[1585]:                   ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
Jul 04 11:23:32 zynthian-rpi5 startx[1585]: IndexError: list index out of range
```
# Configuration
## Hardware
```
Raspberry Pi 5 Model B Rev 1.0
Audio: Dummy device
Display: MIPI DSI 800x480
Wiring: DUMMIES
I2C: Not detected
Profile: v5
```
## System
```
Debian GNU/Linux 12 (bookworm)
Build Date: 2024-05-21
Memory: 9% (734M/8052M)
SD Card: NA (NA/NA)
Temperature: 48.3ºC
Overclock: Medium
```
## MIDI & UI
```
Tuning: 440 Hz
Master Channel: Off
Preload Presets: On
ZS3 (SubSnapShots): On
Power Save: 60 minutes
Audio Levels on Snapshots: On
```
## Software
```
zyncoder: oram (628f3fb) 
zynthian-ui: oram (564a889)
zynthian-sys: oram (4461148)
zynthian-data: oram (e222e11)
zynthian-webconf: oram (f31699f)
```

**Comment by riban-bw** [comment]
This is particularly bad when the last_state snapshot has an uninstalled plugin and Zynthian will not even start. We should put a check to catch this scenario.

**Comment by riban-bw** [comment]
Fixed by commit zynthian-ui 66eb39cb3b45b63981aba43edc759c5225f6f018.
